tools/kernel-build-yml.sh

```diff
@@ -0,0 +1,5 @@
+COMMIT_DATE=$(git log -1 --format=%cd pkg/kernel)
```

Comment:
Presumably this should have # /bin/sh
plus a copyright and SPDX-License identifier comment like other scripts.

Reply:
done.

Reply:
@shjala , I'm just afraid that we can lost the way to differentiate kernels built during development. For example, by using `git log -1` you are not considering changes not committed. I can build one kernel, test, do some changes (and not commit them), build the kernel again, then I will not be able to differentiate these two versions, right? Nevertheless, I don't see problems to keep your approach if we, at least, keep the timestamp (that you are also removing here).

Reply:
@rene 

> you are not considering changes not committed

yes I thought about that before, but my intention is to make it automatic and seamless, otherwise we have to manually set the timestamp string (going with `date` will result in irreproducible builds of the same kernel). 

> I will not be able to differentiate these two versions, right?

not from `cat /proc/version`, but can do with `diff`.

> Nevertheless, I don't see problems to keep your approach if we, at least, keep the timestamp (that you are also removing here).

I'm not sure if I'm getting what you mean by keeping timestamp, would you please elaborate?

Reply:
I meant to not remove the timestamp (as you did by enabling KCONFIG_NOTIMESTAMP). However, I've got your point with this PR. I do understand that you want to generate identical images from identical sources, but I don't like the fact that you are neglecting the changes not committed, because you can generate different kernels with the same version  "tag". Could you consider to check for dirty working trees and mark them as "dirty", for example, a certain kernel version would generate version tag xxx-yyy-zzz, but if you build with modifications not committed, then the tag will be xxx-yyy-zzz-dirty....